docs/src/tutorial.md

```diff
@@ -105,6 +105,52 @@ because `Vector{AbstractFloat}` is a concrete type, whereas `AbstractFloat` is n
 
 If we create `c32 = [1.0f0]` and then `calldouble2(c32)`, we would also see backedges from `double(::Float32)` all the way back to `calldouble2(::Vector{Float32})`.
 
+## A copy-paste analysis of invalidations
+
+The following is a quick "grab and go" script for analyzing invalidations. Insert your code into the `@snoopr` block. The resulting plot shows the
+distributions of the invalidations sorted by the number of children effected. Generally, invalidations with many children matter more than those
+with few children, and thus this shows how many "bad actors" need to be investigated. `show(trees[end])` show the method which leads to the most
+invalidations, with `show(trees[end-1])` being the second most, and so forth.
+
+```julia
+using SnoopCompile
+invalidations = @snoopr begin
+    # Your code goes here!
+    using OrdinaryDiffEq
```

Comment:
Unless there's already a `lorenz` function lying around, `using OrdinaryDiffEq` is the only thing here that can cause invalidations. The rest is a bit misleading because it implies that inference/compilation itself can cause invalidations. (It can't.)

Reply:
There are weird eval cases where it can, like if you change a Plots backend and it evals in some new code. But that might be too rare to specialize on.